tests/performance/load-tests/load-test.sh

```diff
@@ -22,24 +26,34 @@ function print_error() {
   echo -e "${RED}$1${NC}"
 }
 
-function cleanup() {
-  echo "Clean up the environment"
-  kubectl delete dw --all
-  kubectl delete dwt --all
+# Function to display help information
+display_help() {
```

Comment:
Would you keep unified code style and use `function` optional word in here as well?